src/k8s/hack/env.sh

```diff
@@ -11,11 +11,12 @@ REPO_SQLITE="https://github.com/sqlite/sqlite.git"
 REPO_DQLITE="https://github.com/canonical/dqlite.git"
 
 ## Component versions
-TAG_MUSL="v1.2.3"
+TAG_MUSL="v1.2.4"
 TAG_LIBTIRPC="upstream/1.3.3"
-TAG_LIBNSL="v2.0.0"
+TAG_LIBNSL="v2.0.1"
 TAG_LIBUV="v1.48.0"
 TAG_LIBLZ4="v1.9.4"
-TAG_RAFT="v0.18.0"
-TAG_SQLITE="version-3.40.0"
-TAG_DQLITE="v1.15.1"
+TAG_RAFT="v0.18.1"
```

Comment:
are we dropping raft? if so, lets remove this entirely and make it all into a single PR

Reply:
I thought I'd put up a separate PR to drop raft, but I can add it here.